src/main.ts

```diff
@@ -77,4 +77,33 @@ export class Minifier {
       }
     }
   }
+
+  private nextChar(c: string): string { return String.fromCharCode(c.charCodeAt(0) + 1); }
+
+  // Given the last code, returns a string for the new property name.
```

Comment:
I am a bit confused why you called it `lateral`? Isn't this just the next string in an lexicographic (i.e. dictonary-like) ordering.